templates/x86/test/units/mods.config

```diff
@@ -84,6 +84,7 @@ configuration conf {
 	@Runlevel(1) include embox.test.stdlib.strcmp_test
 	@Runlevel(1) include embox.test.stdlib.strlwr_test	
 	@Runlevel(1) include embox.test.stdlib.strcasestr_test	
+	@Runlevel(1) include embox.test.stdlib.strpbrk_test
```

Comment:
Can you split commit for two? one with code and other with adding to template?

Reply:
Split the commit as you asked.